client/components/mma/shared/Card.tsx

```diff
@@ -0,0 +1,42 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { from, palette, space } from '@guardian/source-foundations';
+import type { ReactNode } from 'react';
+
+export const Card = (props: { children: ReactNode }) => {
+	return <div>{props.children}</div>;
+};
+
+Card.Header = (props: {
+	children: ReactNode;
+	backgroundColor?: string;
+	cssOverrides?: SerializedStyles | SerializedStyles[];
```

Comment:
I tend to prefer _not_ allowing arbitrary CSS overrides in components as it can make things more brittle. Any future changes to the component's core styles might conflict with any overrides that are being used elsewhere. It also means a component potentially has infinite variations. That said, it's a pattern that's already used in all of the Source components 😅

It's less of an issue here as we're the only consumer of the component, but once we've built more of the pages it might be nice to see what overrides we've used and add specific variants to the component to handle those cases so we can remove the overrides.

Reply:
I like this 👍